### PR TITLE
update New-NetFirewallRule.md

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2377

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, for your information

Thank you

File: docset/winserver2012-ps/netsecurity/New-NetFirewallRule.md [hunk 1]
```diff
@@ -288,7 +288,7 @@ The acceptable values for this parameter are: Block, Allow, DeferToUser, or Defe
 
 This parameter specifies that traffic that traverses an edge device, such as a network address translation (NAT)-enabled router, between the local and remote computer matches this rule.
 If this parameter is set to DeferToUser or DeferToApp, then Windows allows the user or application to programmatically register with the firewall to receive inbound unsolicited application traffic from the edge device. 
-Note: The DeferToApp and DeferToUser options are only valid for computers running firstref_client_7, firstref_server_7, and Windows Server® 2012.
+Note: The DeferToApp and DeferToUser options are only valid for computers running Windows 7, Windows Server 2008 R2, and Windows Server® 2012.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/netsecurity/New-NetFirewallRule.md:291]:
```suggestion
> [!NOTE]
> The DeferToApp and DeferToUser options are only valid for computers running Windows 7, Windows Server 2008 R2, and Windows Server 2012.
```

File: docset/winserver2012r2-ps/netsecurity/New-NetFirewallRule.md [hunk 2]
```diff
@@ -305,17 +305,14 @@ Specifies that matching firewall rules of the indicated edge traversal policy ar
 
 This parameter specifies how this firewall rule will handle edge traversal cases.
 Valid only when the **Direction** parameter is set to Inbound. 
-The acceptable values for this parameter are: Block, Allow, DeferToUser, or DeferToApp. 
-
+The acceptable values for this parameter are: Block or Allow.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/netsecurity/New-NetFirewallRule.md:308]:
```suggestion
The acceptable values for this parameter are Block and Allow.
```

File: docset/winserver2016-ps/netsecurity/New-NetFirewallRule.md [hunk 3]
```diff
@@ -298,16 +298,14 @@ Accept wildcard characters: False
 Specifies that matching firewall rules of the indicated edge traversal policy are created. 
 This parameter specifies how this firewall rule will handle edge traversal cases.
 Valid only when the *Direction* parameter is set to Inbound. 
-The acceptable values for this parameter are: Block, Allow, DeferToUser, or DeferToApp.
+The acceptable values for this parameter are: Block or Allow.
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/netsecurity/New-NetFirewallRule.md:301]:
```suggestion
The acceptable values for this parameter are Block and Allow.
```

File: docset/winserver2019-ps/netsecurity/New-NetFirewallRule.md [hunk 4]
```diff
@@ -298,16 +298,14 @@ Accept wildcard characters: False
 Specifies that matching firewall rules of the indicated edge traversal policy are created. 
 This parameter specifies how this firewall rule will handle edge traversal cases.
 Valid only when the *Direction* parameter is set to Inbound. 
-The acceptable values for this parameter are: Block, Allow, DeferToUser, or DeferToApp.
+The acceptable values for this parameter are: Block or Allow.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/netsecurity/New-NetFirewallRule.md:301]:
```suggestion
The acceptable values for this parameter are Block and Allow.
```

File: docset/winserver2022-ps/netsecurity/New-NetFirewallRule.md [hunk 5]
```diff
@@ -298,16 +298,14 @@ Accept wildcard characters: False
 Specifies that matching firewall rules of the indicated edge traversal policy are created. 
 This parameter specifies how this firewall rule will handle edge traversal cases.
 Valid only when the *Direction* parameter is set to Inbound. 
-The acceptable values for this parameter are: Block, Allow, DeferToUser, or DeferToApp.
+The acceptable values for this parameter are: Block or Allow.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/netsecurity/New-NetFirewallRule.md:301]:
```suggestion
The acceptable values for this parameter are Block and Allow.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

File: docset/winserver2012r2-ps/netsecurity/New-NetFirewallRule.md [hunk 6]
```diff
@@ -305,17 +305,14 @@ Specifies that matching firewall rules of the indicated edge traversal policy ar
 
 This parameter specifies how this firewall rule will handle edge traversal cases.
 Valid only when the **Direction** parameter is set to Inbound. 
-The acceptable values for this parameter are: Block, Allow, DeferToUser, or DeferToApp. 
-
+The acceptable values for this parameter are Block and Allow.
```
Comment (get-itips) [inline docset/winserver2012r2-ps/netsecurity/New-NetFirewallRule.md:308]: I was able to use "DeferToApp" and "DeferToUser" in WinServ2012R2 (the article you are editing belongs to that version), can you kindly detail what is the reason to remove those values from acceptable list? thanks

File: docset/winserver2016-ps/netsecurity/New-NetFirewallRule.md [hunk 7]
```diff
@@ -298,16 +298,14 @@ Accept wildcard characters: False
 Specifies that matching firewall rules of the indicated edge traversal policy are created. 
 This parameter specifies how this firewall rule will handle edge traversal cases.
 Valid only when the *Direction* parameter is set to Inbound. 
-The acceptable values for this parameter are: Block, Allow, DeferToUser, or DeferToApp.
+The acceptable values for this parameter are Block and Allow.
```
Comment (get-itips) [inline docset/winserver2016-ps/netsecurity/New-NetFirewallRule.md:301]: I was also able to specify DeferToUser and DeferToApp in Server 2016

**Review by get-itips — CHANGES_REQUESTED** [review]
@MaratMussabekov Marat, please review my comments, 2012r2 and 2016 still have those parameters, currently on my lab do not have 2019 and 2022 can you please test those versions? probably the parameters are still there.

**Review by dariomws — CHANGES_REQUESTED** [review]
Agree with @get-itips. I tested both in W2012 R2, 2016 and 2019.

This note/sentence should be removed from all articles of each version.

File: docset/winserver2012-ps/netsecurity/New-NetFirewallRule.md [hunk 8]
```diff
@@ -288,7 +288,8 @@ The acceptable values for this parameter are: Block, Allow, DeferToUser, or Defe
 
 This parameter specifies that traffic that traverses an edge device, such as a network address translation (NAT)-enabled router, between the local and remote computer matches this rule.
 If this parameter is set to DeferToUser or DeferToApp, then Windows allows the user or application to programmatically register with the firewall to receive inbound unsolicited application traffic from the edge device. 
-Note: The DeferToApp and DeferToUser options are only valid for computers running firstref_client_7, firstref_server_7, and Windows Server® 2012.
+> [!NOTE]
```
Comment (get-itips) [inline docset/winserver2012-ps/netsecurity/New-NetFirewallRule.md:291]: Please use a regular Note: as the NOTE tag can't be rendered in get-help output thanks

Comment (JohanFreelancer9) [inline docset/winserver2012-ps/netsecurity/New-NetFirewallRule.md:291]: Note: The DeferToApp and DeferToUser options are only valid for computers running Windows 7, Windows Server 2008 R2, and Windows Server 2012.

Comment (JohanFreelancer9) [inline docset/winserver2012-ps/netsecurity/New-NetFirewallRule.md:291]:
```suggestion
Note: The DeferToApp and DeferToUser options are only valid for computers running Windows 7, Windows Server 2008 R2, and Windows Server 2012.
```

File: docset/winserver2012-ps/netsecurity/New-NetFirewallRule.md [hunk 9]
```diff
@@ -288,7 +288,8 @@ The acceptable values for this parameter are: Block, Allow, DeferToUser, or Defe
...
+> The DeferToApp and DeferToUser options are only valid for computers running Windows 7, Windows Server 2008 R2, and Windows Server 2012.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/netsecurity/New-NetFirewallRule.md:292]:
```suggestion
```

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9 ,
can you please have another look? I applied recommendations from other contributors and also fixed the same error on Set-NetFirewallRule article.
Thank you!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by JohanFreelancer9** [comment]
@MaratMussabekov - looks good.

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Pull request merged**